assets/aml-benchmark/environments/aoai-evaluation/context/Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM mcr.microsoft.com/azureml/openmpi4.1.0-ubuntu20.04:{{latest-image-tag}}
```

Comment:
you should probably try ubuntu22.04, 20.04 will be deprecated in a few months

Reply:
moved to 22.04